content/aro/clf-to-azure/index.md

```diff
@@ -1,31 +1,25 @@
 ---
 date: '2023-03-26T22:07:09.774151'
-title: Using Cluster Logging Forwarder in ARO with Azure Monitor
+title: Using Cluster Logging Forwarder in ARO with Azure Monitor (>=4.13)
 tags: ["ARO", "Azure"]
 authors:
   - Paul Czarkowski
-  - Steve Mirman
-  - Nerav Doshi
-  - Roberto Carratalá
+  - Thatcher Hubbard
 ---
 
-In Azure Red Hat OpenShift (ARO) you can fairly easily set up cluster logging to an in-cluster Elasticsearch using the OpenShift Elasticsearch Operator
-and the Cluster Logging Operator, but what if you want to use the Azure native Log Analytics service?
+> **NOTE**: OpenShift Logging 5.9 supports native forwarding to Azure Monitor and Azure Log Analytics, which is available on clusters running OpenShift 4.13 or higher. For clusters running OpenShift 4.12 or earlier, see the [legacy setup document here](/experts/aro/clf-to-azure-old/) for help with configuration.
 
-There's a number of ways to do this, for example installing agents onto the VMs (in this case, it would be a DaemonSet with hostvar mounts) but that isn't ideal in a managed system like ARO.
-
-Fluentd is the log collection and forwarding tool used by OpenShift, however it does not have native support for Azure Log Analytics. However Fluent-bit which supports many of the same protocols as Fluentd does have [native support](https://docs.fluentbit.io/manual/pipeline/outputs/azure) for Azure Log Analytics.
-
-Armed with this knowledge we can create a fluent-bit service on the cluster to accept logs from fluentd and forward them to Azure Log Analytics.
+If you're running Azure Red Hat OpenShift (ARO), you may want to be able to view and query the logs the platform and your workloads generate in Azure Monitor. With the release of the Cluster Logging Operator version 5.9, this can be done in a single step with some YAML configuration.
 
 ## Prepare your ARO cluster
 
 1. [Deploy](/experts/quickstart-aro) an ARO cluster
 
+1. Follow the OpenShift documentation for [installing the OpenShift Logging Operator](https://docs.openshift.com/container-platform/4.14/logging/cluster-logging-deploying.html) for your version of OpenShift. These instructions cover the various methods (CLI, Web Console) of installation.
```

Comment:
The reason I did this is because there are multiple options and writing ones with a specific version in them felt like inviting bit-rot.  The official docs for the install aren't that bad are they? I can replicate the stuff from your original doc if we think that's the right way to go.